Action connectors can be imported/exported with rules (#3021)
elastic/security-docs: docs/detections/api/rules/rules-api-import.asciidoc

NOTE: This API supports {kibana-ref}/api.html#token-api-authentication[Token-based authentication] only.

> NOTE: To import rules with actions, you need at least `Read` privileges for the `Action and Connectors` feature. To overwrite existing connectors or add new connectors, you need `All` privileges for the `Action and Connectors` feature. If you're importing rules without actions, `Action and Connectors` feature privileges are not required. Refer to <<enable-detections-ui>> for more information.

Reviewer: ```suggestion
NOTE: To import rules with actions, you need at least `Read` privileges for the `Action and Connectors` feature. To overwrite or add new connectors, you need `All` privileges for the `Actions and Connectors` feature. To import rules without actions,  you don't `Actions and Connectors` privileges. Refer to <<enable-detections-ui>> for more information.
```

Author: ```suggestion
NOTE: To import rules with actions, you need at least `Read` privileges for the `Action and Connectors` feature. To overwrite or add new connectors, you need `All` privileges for the `Actions and Connectors` feature. To import rules without actions,  you don't need `Actions and Connectors` privileges. Refer to <<enable-detections-ui>> for more information.
```

[IMPORTANT]
=================
Rule actions and connectors are included in the exported file, but sensitive information about the connector (such as authentication credentials) _is not_ included. You must re-add missing connector details after importing detection rules.

> You can use {kib}'s {kibana-ref}/managing-saved-objects.html#managing-saved-objects-export-objects[Saved Objects] UI (*Stack Management* -> *Kibana* -> *Saved Objects*) or you can use the Saved Objects APIs (experimental) to {kibana-ref}/saved-objects-api-export.html[export] and {kibana-ref}/saved-objects-api-import.html[import] any necessary connectors before importing detection rules.

Reviewer 2: Nit:
```suggestion
You can use {kib}'s {kibana-ref}/managing-saved-objects.html#managing-saved-objects-export-objects[Saved Objects] UI (*Stack Management* -> *Kibana* -> *Saved Objects*) or the Saved Objects APIs (experimental) to {kibana-ref}/saved-objects-api-export.html[export] and {kibana-ref}/saved-objects-api-import.html[import] any necessary connectors before importing detection rules.
```
